settings/settings.go

```diff
@@ -590,3 +590,70 @@ func jsonString(conn *Connection) string {
 	}
 	return extraString
 }
+
+func FileToConnectionYAML(settingsFile string) error {
```

Comment:
```suggestion
func WriteAirflowSettingstoYAML(settingsFile string) error {
```
nit: can we get to a more appropriate name for the function, as of now the name is somewhat misleading

Reply:
okay I'll change